src/web/theme/pages/TryIt/SelectStore/SelectStore.js

```diff
@@ -24,83 +26,75 @@ const SelectStore = ({
-            <Select
-              value={
-                loading
-                  ? null
-                  : addresses.filter(selectAddress => {
-                      return selectAddress.value === address;
-                    })
-              }
-              onChange={value => {
-                setStepIsFilled(true);
-                setTryItState({ address: value.value });
-              }}
-              options={loading ? [{ label: "", value: "" }] : addresses}
-            />
-          </div>
-          <div className="select-store__map">
-            {loading ? (
-              <div>Loading...</div>
-            ) : (
-              <Map
-                center={[43.584296, 1.44182]}
-                zoom={14}
-                style={{ height: "600px", width: "800px", overflow: " hidden" }}
-              >
-                <TileLayer
-                  attribution="&amp;copy <a href=&quot;http://osm.org/copyright&quot;>OpenStreetMap</a> contributors"
-                  url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-                />
-                {stores.map(store => {
-                  return (
-                    <Marker
-                      key={store.name}
-                      position={[
-                        loading ? 0 : store.coordinates.latitude,
-                        loading ? 0 : store.coordinates.longitude
-                      ]}
-                      icon={
-                        new Icon.Default({
-                          imagePath: "/static/media/",
-                          iconUrl: `${icon.replace(/\/static\/media\//, "")}`,
-                          shadowUrl: `${shadow.replace(
-                            /\/static\/media\//,
-                            ""
-                          )}`
-                        })
-                      }
-                      onClick={() => {
-                        setStepIsFilled(true);
-                        setTryItState({ address: store.address });
-                      }}
-                    >
-                      <Popup>
-                        <div>
-                          <div>Store name ─ {loading ? null : store.name}</div>
-                          <div>Email: {loading ? null : store.owner.email}</div>
-                          <div>Phone: {loading ? null : store.phone}</div>
-                        </div>
-                      </Popup>
-                    </Marker>
-                  );
-                })}
-              </Map>
-            )}
-          </div>
-        </Fragment>
-      ) : null}
+      <Step
+        currentStep={currentStep}
+        stepIndex={getStepIndex(stepName)}
+        actionDescription="Select a store location on the map"
+        collapsed={collapsed}
+      >
+        <div className="select-store__searchbar">
+          <Select
+            value={
+              loading
+                ? null
+                : addresses.filter(selectAddress => {
```

Comment:
Seems that you can directly use `address` as value

Reply:
nope.